package.json

```diff
@@ -109,6 +109,7 @@
         "react-dom": "16.8.6",
         "react-icons": "3.7.0",
         "react-router-dom": "4.3.1",
+        "regenerator-runtime": "0.13.3",
```

Comment:
Seems like `regenerator-runtime` is being used only for testing, might be makes sense to move it to `devDependensies`?

Reply:
@antipin it is in devDependencies already

Reply:
Ah.. Sorry, I was confused by `react-dom`,  `react-icons` and `react-router-dom`, that looks like non-dev deps.